### ISSUE TITLE
[feature-request]: Have default color theme be colour-blind friendly

### ISSUE THREAD

**Issue by lwjohnst86** [seedcase-project/design#59]
### What feature would you like to request?

Accessibility is an important topic, one of which is colour themes that work for those with color blindness (of some degree). The default for anything we make should be colour-blind friendly.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by lwjohnst86** [comment]
As I'm moving the colour scheme over to the websites, I realise our current colour scheme isn't very colour blind friendly, especially in relation to the complimentary "purple" colour. This complimentary colour choice is important for contrasting things like links. There are a few websites, like
[this](https://mk.bcgsc.ca/colorblind/palettes.mhtml#page-container) and [this](https://davidmathlogic.com/colorblind/#%23D81B60-%231E88E5-%23400000-%23641862-%23196440) that get into and show some good colours to use. I'll make use of what I can, but this will probably have to take a bit more thought and work.

**Comment by K-Beicher** [comment]
@signekb did you say you had some links to websites detailing colourblind friendly website building? If so, could you add the links here?

**Comment by signekb** [comment]
I haven't used any for website building, but I used [this one](https://davidmathlogic.com/colorblind) when I created the colour theme for visualisations in my thesis (and assignments in general) at uni
